### PR TITLE
Add a new dashboard page with ability to create new specifications

### PR DESCRIPTION
**Pull request by LornaHarwood**
## Changes in this PR

After sign in, users are now sent to the Specification Dashboard (instead of starting a new spec), from where they can choose to create a new specification.

A subsequent PR will add the ability to view existing specifications.

## Screenshots of new UI

localhost_3000_dashboard (1)

### CODE REVIEW

**Review by tahb — COMMENTED** [review]
Looks good to me.

I notice that when we click the link to start a journey that there's no way back to the dashboard to start again. Before this we were able to click the service name in the header to get back but now we have a new page in the journey that requires 2 clicks to get back to. Should we add some more back navigation, what do you think?

File: app/views/journeys/show.html.erb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <%= content_for :title, @journey.category.capitalize %>
 
+<%= link_to I18n.t("generic.button.back"), dashboard_path, class: "govuk-back-link" %>
```
Comment (tahb) [inline app/views/journeys/show.html.erb:3]: Should we have a feature test for this?

Comment (LornaHarwood) [reply app/views/journeys/show.html.erb:3]: I have added a test for this within the `anyone_can_view_a_list_of_tasks_spec`, as this feels like the most logical place to put it

**Review by tahb — APPROVED** [review]
LGTM

**Pull request merged**